docs/glossary/README.md

```diff
@@ -396,9 +400,12 @@ In OpenProject, a wiki is defined as a [module](#module) that allows to use wiki
 
 ![A wiki module in OpenProject](glossary-openproject-wiki.png)
 
-### Work and Remaining Work
+### Work, Remaining Work and % Complete
+
+In OpenProject, '**Work**' refers to a work package attribute indicating the estimated hours needed to complete a task.  
+'**Remaining work**' is a work package attribute that shows how much work is left to finish the work package. It is automatically calculated if you work with status-based progress reporting. And '**% Complete**'shows an automatically calculated work package attribute that shows in percentage how much work is already completed. All three attributes are important for [progress reporting with OpenProject](https://www.openproject.org/blog/changes-progress-work-estimates/).
```

Comment:
I would add a link for "status-based progress reporting" to https://www.openproject.org/docs/user-guide/time-and-costs/progress-tracking/#status-based-progress-reporting

And `And '**% Complete**'shows an automatically calculated work package attribute that shows in percentage ...` feels wrong. What about `And '**% Complete**' is an automatically calculated work package attribute that shows in percentage ...`?

Reply:
Maybe "It is automatically calculated if you work with status-based progress reporting." could even be removed. It's a detail and it is better explained in "progress reporting with OpenProject" page.

Reply:
Thanks a lot!